src/Service.Tests/SqlTests/RestApiTests/Find/FindApiTestBase.cs

```diff
@@ -127,6 +127,22 @@ public virtual async Task FindOnTableWithUniqueCharacters()
                 sqlQuery: GetQuery("FindOnTableWithUniqueCharacters"));
         }
 
+        /// <summary>
+        /// Tests the Rest Api to validate that queries work
+        /// when there is the same table name in two different
+        /// schemas.
+        /// </summary>
+        /// <returns></returns>
+        [TestMethod]
+        public virtual async Task FindOnTableWithNamingCollision()
```

Comment:
what is the expected behavior of "FindOnTableWithNamingCollision" 
-> (Actionable Exception/error communicated to user) error is caught and properly messaged to user? 
-> (Gracefully handled) no error because you now use the schema name when making comparisons?

Reply:
This test validates that the query works despite there being a naming collision, so gracefully handled now because we are using the schema.